docs/user-guide/mvd-editor.md

```diff
@@ -2,40 +2,71 @@
+1. From the **File** menu, select Open Datasets. You can also use (ALT+K).
+2. In the Dataset field, specify the name of the dataset you want to open. 
+3. Click **Open** 
+
+## Deleting a dataset
+
+To delete a dataset, follow these steps:
+
+1. In the file tree, right-click on a dataset you want to delete.
+2. From the right-click menu, click **Delete**. A warning dialogue will appear. 
+3. Click **Delete** 
 
 ## Opening a directory
 
 To open a directory on the system, complete these steps:
 
-1. From the **File** menu, select **Open Directory**.(Alternatively, you can click **Open Directory** in the File Explorer.)
-
-2. From the **Open Directory**, **Input Your Directory** field, type the name of the directory you want to open. For example: `/u/zs1234`
+1. From the **File** menu, select **Open Directory**. You can also use (ALT+O).
+2. In the Directory field, specify the name of the directory you want to open. For example: `/u/zs1234`
 3. Click **Open**.
 
 The File Explorer on the left side of the window lists the folders and files in the specified directory. Clicking on a folder expands the tree. Clicking on a file opens a tab that displays the file contents.
 
+## Creating a new directory
+
+1. Right-click on a location in the directory tree where you want to create a new directory.
+2. From the right-click menu, click **Create a directory...**.
+3. Specify a directory name in the Directory Name field. 
+4. The Path will be set to the location that you intially right-clicked to open the dialogue. You can specify a different location in the Path field. 
+5. Click **Create**    
+
 ## Creating a new file
 
 To create a new file, complete these steps:
 
-1. From the **File** menu, select **New File**. The **New File** tab opens.
-2. From the **New File**, **File Name** field, type the name of the file.
-3. Click **Create**.
+1. From the **File** menu, select **New File**. You can also use (ALT+N).
+2. From the **File** menu, select **Save** to save the newly created file. You can also use (Ctrl+S)
+3. In the File Name field, specify the file name for the newly created file.
+4. Choose an encoding option from the Encoding dropdown menu. The directory will be prefilled if you are creating the new file in an existing folder.
+5. Click **Save**
+   
 
 ## Saving a file
 
 To save a file, click **File** > **Save**.
 
-**Note:** To save all files, click **File** > **Save All** (or Ctrl+S).
\ No newline at end of file
+## Hotkeys
+
+- Shift TAB: Cycle through the menu bar, browsing type, search bar, file tree, and editor component.  
+    - Individual options within the menu bar and individual nodes within the file tree can be navigated with the arrow keys and ENTER (to select).
+
+|Hot Key|Command|
+|---	|---	|
+|ALT+K  |Open a dataset |
+|ALT+O  |Open a directory |
+|ALT+N  |Create a new file |
+|CTRL+S |Save file |
+|ALT+Shift+M |Navigate Menu bar (use arrow keys) |
+|ALT+Shift+S   	|Search bar |
+|ALT+Shift+1   	|Editor Component |
```

Comment:
![image](https://user-images.githubusercontent.com/20528015/84698783-66650e80-af1e-11ea-9097-24384dc47b1d.png)

Reply:
Alt+W to close a file